tests/dummy/app/templates/layout/introduction.hbs

```diff
@@ -0,0 +1,265 @@
+    with our simple layout class convention. The alias is simply used as suffix extensions to the base attribute class.
+  </p>
+  <p>
+    This notation results in, for example, the following table for the layout and flex classes:
+  </p>
+
+  <table class="paper-api">
+    <thead>
+      <tr>
+        <th>layout class</th>
+        <th>flex class</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><strong>layout</strong></td>
+        <td><strong>flex</strong></td>
+        <td>Sets the default layout direction &amp; flex unless overriden by another breakpoint.</td>
+      </tr>
+      <tr>
+        <td><strong>layout-xs</strong></td>
+        <td><strong>flex-xs</strong></td>
+        <td>width &lt; <strong>600</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-gt-xs</strong></td>
+        <td><strong>flex-gt-xs</strong></td>
+        <td>width &gt;= <strong>600</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-sm</strong></td>
+        <td><strong>flex-sm</strong></td>
+        <td>width <strong>600</strong>px &lt;= width &lt; <strong>960</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-gt-sm</strong></td>
+        <td><strong>flex-gt-sm</strong></td>
+        <td>width &gt;= <strong>960</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-md</strong></td>
+        <td><strong>flex-md</strong></td>
+        <td><strong>960</strong>px &lt;= width &lt; <strong>1280</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-gt-md</strong></td>
+        <td><strong>flex-gt-md</strong></td>
+        <td>width &gt;= <strong>1280</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-lg</strong></td>
+        <td><strong>flex-lg</strong></td>
+        <td><strong>1280</strong>px &lt;= width &lt; <strong>1920</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-gt-lg</strong></td>
+        <td><strong>flex-gt-lg</strong></td>
+        <td>width &gt;= <strong>1920</strong>px</td>
+      </tr>
+      <tr>
+        <td><strong>layout-xl</strong></td>
+        <td><strong>flex-xl</strong></td>
+        <td>width &gt;= <strong>1920</strong>px</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <p>Below is an example of the usage of the responsive layout classes:</p>
+
+  {{#paper-card as |card|}}
+  {{! BEGIN-SNIPPET layout-introduction }}
+  <div class="layout-column example introduction-example">
+    <div class="flex-33 flex-md-{{box1Height}} box-one"></div>
+    <div class="flex-33 layout-{{direction}} layout-md-row box-two">
+      <div class="flex-20 flex-md-10 hide-lg box-two-one"></div>
+      <div class="flex-30 {{if hideBox 'hide-md'}} flex-md-25 box-two-two"></div>
+      <div class="flex-20 flex-md-65 box-two-three"></div>
+    </div>
+    <div class="flex box-three"></div>
+  </div>
+  {{! END-SNIPPET }}
+  {{#card.content}}
```

Comment:
Can we put the controls here? It gets hard to see the changes we're making because the controls are too far down (I'm on a small laptop).